CHANGELOG.md

```diff
@@ -15,35 +15,35 @@ and includes an additional section for migration notes.
 
 ## [Unreleased]
 
+- *Migration Notes* - Notes for end users to migrate to the version.
```

Comment:
Should be headdings L3 for all of these
like the following
`- *Migration Notes* - Notes for end users to migrate to the version.` becomes
`### Migration Notes`

Reply:
Fixed.